### DISCUSSION TITLE
Unable to open for fetching projection

### DISCUSSION THREAD

**Discussion by Antguz** [davidfrantz/force#326] [Q&A] [answered]
Dear community,

I am trying to ingest new imagery from Landsat 8/9 to the L2-ARD on top of the existing one, and I am getting an error that I have never seen that says 'unable to open for fetching projection' in all the log files. 

I first look at the projection and `gdalsrsinfo` says that the projection is validated with success. Going deeper, this error shows up in the new imagery, not in the imagery of two years ago that I used before. 

Any idea what is happening? I can share imagery and parameters files if required. I am running the last version of force though singularity.

Regards.

AG.

**Comment by davidfrantz** [comment]
Hi AG,

I have never seen this message in the field ;) but it gets emitted when evaluating the metadata of the input images.

I guess it either means that your input images don't have projection metadata (unlikely), or that something is wrong with the file structure? Wrong path given or incomplete download?

Best, 
David

**Reply by Antguz** [reply, marked as answer]
I guest, it was my mistake. I was using level2 imagery. My bad ...
